Dockerfile

```diff
@@ -1,13 +1,17 @@
-FROM python:3.6-alpine
+FROM pypy:3.9-slim-buster
```

Comment:
Hi @mirceaulinic 
you mentioned pypy has a better performance. Is it build or running performance?

Reply:
hi guys, i got `AttributeError: cython_sources` when building the dockerfile which is related to https://github.com/yaml/pyyaml/issues/601. IRRD has a fix (https://github.com/irrdnet/irrd/pull/816) for this but not released. I tried to pip install main branch but got other errors. Any ideas? Thanks